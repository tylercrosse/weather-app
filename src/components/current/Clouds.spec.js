import React from 'react';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";
import Clouds from './Clouds';

const setup = () => {
  const props = {
    classes: {}
  }

  const component = <Clouds {...props} />;
  const wrapper = shallow(component);

  return {
    props,
    wrapper
  }
}

describe('<Clouds />', () => {
  it('should render correctly', () => {
    const { wrapper } = setup();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})
