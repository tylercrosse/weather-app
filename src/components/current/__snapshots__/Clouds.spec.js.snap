// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Clouds /> should render correctly 1`] = `
<div
  className="clouds"
>
  <svg
    class="cloud cloud__back undefined"
    viewBox="0 0 1000 100"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0,0V49.92C24.95,79.85,71.64,100,125.12,100S225.3,79.85,250.24,49.92C275.19,79.85,321.88,100,375.36,100S475.54,79.85,500.48,49.92C525.43,79.85,572.12,100,625.6,100c53.15,0,99.58-19.9,124.64-49.52C775.3,80.1,821.73,100,874.88,100c53.49,0,100.17-20.15,125.12-50.08V0Z"
    />
  </svg>
  <svg
    class="cloud cloud__middle undefined"
    viewBox="0 0 1000 100"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0,0V49.92C24.95,79.85,71.64,100,125.12,100S225.3,79.85,250.24,49.92C275.19,79.85,321.88,100,375.36,100S475.54,79.85,500.48,49.92C525.43,79.85,572.12,100,625.6,100c53.15,0,99.58-19.9,124.64-49.52C775.3,80.1,821.73,100,874.88,100c53.49,0,100.17-20.15,125.12-50.08V0Z"
    />
  </svg>
  <svg
    class="cloud cloud__front undefined"
    viewBox="0 0 1000 100"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0,0V49.92C24.95,79.85,71.64,100,125.12,100S225.3,79.85,250.24,49.92C275.19,79.85,321.88,100,375.36,100S475.54,79.85,500.48,49.92C525.43,79.85,572.12,100,625.6,100c53.15,0,99.58-19.9,124.64-49.52C775.3,80.1,821.73,100,874.88,100c53.49,0,100.17-20.15,125.12-50.08V0Z"
    />
  </svg>
</div>
`;
